src/helpers/bumpVersion.js

```diff
@@ -1,3 +1,6 @@
+const core = require('@actions/core')
+const semverValid = require('semver').valid
```

Comment:
Just noticed that `semver` is not in our package.json, could we add it there?

Reply:
done in https://github.com/TriPSs/conventional-changelog-action/pull/50/commits/283100b0c8b51e728cf3c1553169172ae432b703